transforms/csdl/preprocess_csdl.xsl

```diff
@@ -171,6 +171,30 @@
         </xsl:copy>
     </xsl:template>
 
+    <!-- Add Acceptable Media Types -->
+    <xsl:template match="edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='organizationalBrandingProperties']/edm:Property[@Name='backgroundImage']|
+                         edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='organizationalBrandingProperties']/edm:Property[@Name='bannerLogo']|
+                         edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='organizationalBrandingProperties']/edm:Property[@Name='customCSS']|
+                         edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='organizationalBrandingProperties']/edm:Property[@Name='favicon']|
+                         edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='organizationalBrandingProperties']/edm:Property[@Name='headerLogo']|
+                         edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='organizationalBrandingProperties']/edm:Property[@Name='squareLogo']|
+                         edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='organizationalBrandingProperties']/edm:Property[@Name='squareLogoDark']">
+        <xsl:copy>
+            <xsl:copy-of select="@* | node()" />
+            <Annotation Term="Org.OData.Core.V1.AcceptableMediaTypes">
+                <Collection>
+                    <String>image/bmp</String>
+                    <String>image/jpg</String>
+	                <String>image/jpeg</String>
```

Comment:
```suggestion
	            <String>image/jpeg</String>
```

Reply:
I promise they are aligned 😄, view the PR on the github.dev web-based editor (press the . key)